content/backstage/plugins/github-insights.md

```diff
@@ -25,25 +25,24 @@ gettingStarted: # What will this step accomplish?
     language: typescript
     code: |
       // packages/app/src/plugins.ts
-      export { plugin as GitHubInsights } from '@roadiehq/backstage-plugin-github-insights';
```

Comment:
I'm 90% sure that this plugin has been upgraded and this step shouldn't be needed any more. Perhaps the plugin README just wasn't updated?

Reply:
@dtuite You are right, I will also correct it in README

Reply:
Cool. Other than that this is looking pretty good.